src/Services/Air/AirParser.js

```diff
@@ -755,14 +761,18 @@ function extractBookings(obj) {
         (key) => {
           const pricingInfo = booking['air:AirPricingInfo'][key];
 
-          const uapiSegmentRefs = pricingInfo['air:BookingInfo'].map(
+          const uapiSegmentRefs = (pricingInfo['air:BookingInfo'] || []).map(
             segment => segment.SegmentRef
           );
 
           const uapiPassengerRefs = pricingInfo[`common_${this.uapi_version}:BookingTravelerRef`];
 
           const fareInfo = pricingInfo['air:FareInfo'];
 
+          if (!fareInfo) {
```

Comment:
Is it fine that taxes, passengers won't be parsed when fare info is missing. It seem they are parsed from another structure

Reply:
should be ok now